doc/guide/introduction/tutorial.xml

```diff
@@ -43,13 +45,16 @@
           software for version management.
         </p>
         <p>
-          There are other similar products such as
-          <url href="https://about.gitlab.com/">GitLab</url> for managing <pretext/> documents,
-          as well as other ways to write <pretext/> that don't require anything
-          besides installing the free and open-source <pretext/> software onto your own device.
-          However, we will use GitHub for this tutorial as it the most popular way to share and disseminate <pretext/>
-          documents, and provides the easiest pathway to getting started to actually write in the <pretext/>
-          language.
+          There are other services such as <url href="https://cocalc.com">CoCalc</url> and
```

Comment:
I still think the CoCalc instructions from earlier could be linked to - do they still exist elsewhere in the guide?  I feel like maybe they do - maybe https://pretextbook.org/doc/guide/html/topic-build-in-cocalc.html ?

Reply:
> Using CoCalc. CoCalc has graciously already installed the PreTeXt-CLI, as well as LATEX, so there is almost no setup required, although the version available there might lag behind the most recent release. Simply visit [cocalc.com](https://cocalc.com/). There you can click on the “Run CoCalc Now” button, or sign up for a free account so you can save your work. Once you are in a CoCalc project, click on the “New” tab and select the “Linux terminal” button. This will give you a command prompt where you can start typing commands. At this point, you can skip directly to the next section of the tutorial, [Section 2.2](https://pretextbook.org/doc/guide/html/quickstart-example-projects.html).

This is what we had - I'll add an xref to https://pretextbook.org/doc/guide/html/topic-build-in-cocalc.html which I think is sufficient.